app/helpers/measures_helper.rb

```diff
@@ -21,4 +23,24 @@ def legal_act_regulation_url_link_for(measure)
 
     sanitize(link, attributes: %w[href target role rel class title])
   end
+
+  def check_how_to_export_goods_link(commodity_code:, country_code:, country_name:, eu_member:)
+    if eu_member
+      cheg_commodity_code = commodity_code_for_cheg_service(commodity_code)
+
+      link_to("Check how to export #{commodity_code} to #{country_name} (link opens in new tab)",
+              "#{CHECK_DUTIES_SERVICE_URL}/summary?d=#{country_code}&ds=gtp&tab=tree&pc=#{cheg_commodity_code}",
+              target: '_blank', rel: 'noopener')
+    else
+      link_to('Check how to export commodity goods (link opens in new tab)', CHECK_DUTIES_SERVICE_URL, target: '_blank', rel: 'noopener')
+    end
+  end
+
+  private
+
+  # Right-Strip of the zeros in pairs.
+  # Example: 8514101000 -> 85141010
+  def commodity_code_for_cheg_service(commodity_code)
```

Comment:
So the commodity knows about the format of its own commodity code. I think we should move this to the model which is the single source of truth for how the commodity code gets presented. It's also easier to decouple from the link and test in isolation

Reply:
yeah I agree 100%,
I've moved the test too.